go/flags/endtoend/vttestserver.txt

```diff
@@ -128,9 +127,6 @@ Usage of vttestserver:
       --topo_etcd_tls_ca string                                          path to the ca to use to validate the server cert when connecting to the etcd topo server
       --topo_etcd_tls_cert string                                        path to the client cert to use to connect to the etcd topo server, requires topo_etcd_tls_key, enables TLS
       --topo_etcd_tls_key string                                         path to the client key to use to connect to the etcd topo server, enables TLS
-      --topo_global_root string                                          the path of the global topology data in the global topology server
-      --topo_global_server_address string                                the address of the global topology server
-      --topo_implementation string                                       the topology implementation to use
```

Comment:
I'd expect vttestserver to have all the same flags as vttablet + vtgate + vtctld.
So it doesn't look right to remove these here.

Reply:
or maybe they are hard-coded?

Reply:
Vttestserver spwans the vtcombo process, but for whatever reason it uses flags `external_topo_implementation`, etc. It passes these values along to vtcombo as `topo_implementation` etc

Reply:
Weird I know, but it doesn't use these `topo_implementation` flags itself.